foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/internal/localization/i18n/LoggingLocalizationResource.java

```diff
@@ -483,16 +483,6 @@ Current thread (id: {3} name: {4}) stack trace:
         // JPA 3.2
         { "unknown_property_type", "Unknown {0} type of {1} persistence property"},
         { "error_queryTimeoutParse", "Cannot parse the {0} jakarta.persistence.query.timeout property value: {1}"},
-        { "schema_default_truncate_tables_failed", "Failed to truncate tables in the default table schema: {0}"},
-        { "schema_default_create_tables_failed", "Failed to create tables in the default table schema: {0}"},
-        { "schema_default_drop_tables_failed", "Failed to drop tables in the default table schema: {0}"},
-        { "schema_default_replace_tables_failed", "Failed to replace tables in the default table schema: {0}"},
-        { "schema_default_extend_tables_failed", "Failed to extend tables in the default table schema: {0}"},
-        { "schema_drop_object_failed", "Failed to drop object in the default table schema: {0}"},
-        { "validate_object_space", "validate object space." },
-        { "stack_of_visited_objects_that_refer_to_the_corrupt_object", "stack of visited objects that refer to the corrupt object: {0}" },
-        { "corrupt_object_referenced_through_mapping", "The following corrupt object is referenced through mapping: {0}" },
-        { "corrupt_object", "corrupt object: {0}" },
```

Comment:
Is it this removal correct? E.g. I see `validate_object_space` usage in `org.eclipse.persistence.internal.sessions.UnitOfWorkImpl` .

Reply:
```
log(SessionLog.FINER, SessionLog.TRANSACTION, "validate_object_space");
```
Yes, there are just 2 usages in `UnitOfWorkImpl`, both with `FINER` level so think it's ok.